plugins/Dashboard/javascripts/widgetMenu.js

```diff
@@ -25,7 +25,7 @@ widgetsHelper.getAvailableWidgets = function () {
                 widgetsHelper.availableWidgets = data;
             }
         );
-        ajaxRequest.send(true);
+        ajaxRequest.send();
```

Comment:
Just FYI: I started refactoring this one a week or two weeks ago and noticed there's definitely some work to do to make it work. Eg this method https://github.com/piwik/piwik/blob/2.14.0-b10/plugins/Dashboard/javascripts/widgetMenu.js#L41 and this method https://github.com/piwik/piwik/blob/2.14.0-b10/plugins/Dashboard/javascripts/widgetMenu.js#L60 accesses it in a sync way but those methods and all callers need to be refactored to work async